news/2025/2025-01-12-tournament-committee-applications-now-open.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Tournament Committee applications now open!
+date: 2025-01-12 15:00:00 +0000
+---
+
+Do you want to help shape the future of the osu! tournament community? If so, then apply now to join the [Tournament Committee (TC)](/wiki/People/Tournament_Committee) during our first ever open application cycle!
+
+We are seeking passionate individuals with significant tournament experience—including players—to help support and improve the community.
+
+## What Does the Tournament Committee Do?
+
+The TC plays a vital role in maintaining and supporting the growth of the tournament scene. As a member, your primary responsibilites will include:
+
+- **Tournament Reviews**: Approving or denying badge support for tournaments and communicating any necessary changes to organisers.
```

Comment:
we put the ":" inside the bolded text in similar lists

Reply:
Fixed